docs/installation/sources/sources_linux.rst

```diff
@@ -171,7 +171,7 @@ Also add the `Gtest repository <https://github.com/google/googletest>`_ into the
 
 .. code-block:: bash
 
-    git clone https://github.com/google/googletest src/googletest-distribution
+    git clone --branch release-1.12.0 https://github.com/google/googletest src/googletest-distribution
```

Comment:
I'd use 1.11, it's what we use on [Jenkins](https://gitlab.intranet.eprosima.com/eProsima/fastrtps-ci/-/blob/master/depends.repos). Applies to all
```suggestion
    git clone --branch release-1.11.0 https://github.com/google/googletest src/googletest-distribution
```

Reply:
FYI, `1.12.0` is the version used in eProsima CI.

Reply:
We'll need to upgrade at some point then, thanks!